csharp/OnnxRuntime.CSharp.proj

```diff
@@ -8,10 +8,19 @@ CMake creates a target to this project
 <Project DefaultTargets="Build">
   
   <PropertyGroup>
+    <OrtPackageId Condition=" '$(OrtPackageId)' == '' ">Microsoft.ML.OnnxRuntime</OrtPackageId>
```

Comment:
WIll be overridden for other packages (.Gpu, .MKLML, etc.)

Reply:
I don't see a test run for the Gpu and MKLML pipelines. Would it be possible to run this branch against the other two pipelines?

Reply:
Yes, running them now :). I needed corresponding YAML changes for those pipelines which I didn't have previously. I have made them now and kicked those pipelines off.